### PR TITLE
T20768

### PR DESCRIPTION
**Pull request by ebassi**
https://phabricator.endlessm.com/T20768

### CODE REVIEW

File: tools/picard.in [hunk 1]
```diff
@@ -15,6 +15,10 @@ if [ "$RUN_DEBUG" != "" ]; then
     DEBUG_COMMAND="gdb --args"
 fi
 
+if [ "$RUN_PROFILE" != "" ]; then
+    export EOS_PROFILE=1
+fi
```
Comment (tchx84) [inline tools/picard.in]: Picard doesn't use EknContent nor make_ready probes (uses a dummy engine).

File: tools/kermit.in [hunk 2]
```diff
@@ -15,6 +15,10 @@ if [ "$RUN_DEBUG" != "" ]; then
     DEBUG_COMMAND="gdb --args"
 fi
 
+if [ "$RUN_PROFILE" != "" ]; then
+    export EOS_PROFILE=1
+fi
```
Comment (tchx84) [inline tools/kermit.in]: These tools don't use Engine/Domain interface, so this won't do anything.

File: tools/picard.in (hunk 1 above)
Comment (ebassi) [reply tools/picard.in]: It's entirely possible that the probes are not tripped directly; I'm going through the code to check.

**Comment by ebassi** [comment]
In the end, I've removed the wrapper from kermit and picard: there's nothing really to probe in the code paths that are used by these two tools, and all I could get were 1 sample probes.

**Pull request merged**